README.md

```diff
@@ -1,163 +1,53 @@
-npm(1) -- a JavaScript package manager
-==============================
+[![GitHub Workflow Status (branch)](https://img.shields.io/github/workflow/status/npm/cli/Node%20CI/latest)](https://github.com/npm/cli/actions?query=workflow%3A%22Node+CI%22+branch%3Alatest) [![Coveralls github branch](https://img.shields.io/coveralls/github/npm/cli/latest)](https://coveralls.io/github/npm/cli?branch=latest)
 
-[![Build Status](https://img.shields.io/travis/npm/cli/latest.svg)](https://travis-ci.org/npm/cli)
+# npm - a JavaScript package manager
 
-## SYNOPSIS
+### Requirements
 
-This is just enough info to get you up and running.
+* [**Node.js** `v10`](https://nodejs.org/en/download/) or higher must be installed to run this program
 
-Much more info will be available via `npm help` once it's installed.
+### Installation
 
-## IMPORTANT
+**`npm`** comes bundled with [**`node`**](https://nodejs.org/), & most third-party distributions, by default. Officially supported downloads/distributions can be found at: [nodejs.org/en/download](https://nodejs.org/en/download)
 
-**You need node v6 or higher to run this program.**
+#### Direct Download
 
-To install an old **and unsupported** version of npm that works on node v5
-and prior, clone the git repo and dig through the old tags and branches.
+You can download & install **`npm`** directly from [**npmjs**.com](https://npmjs.com/) using our custom `install.sh` script:
```

Comment:
does this still work? nvm only used it for ancient node versions, but it stopped working for that a long time ago.

Reply:
it should! i cleaned it up a few months ago and have used it for testing in docker containers and vms

Reply:
Ooh - if there's any way it can be given an env var to override which version of npm it installs, that would make it possible for me to fix nvm's usage of it :-D